dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -371,13 +372,11 @@ control dash_ingress(
             set_dst_tag;
         }
     }
-
+    
     apply {
-
         /* Send packet on same port it arrived (echo) by default */
-#ifdef TARGET_BMV2_V1MODEL
-        standard_metadata.egress_spec = standard_metadata.ingress_port;
-#endif // TARGET_BMV2_V1MODEL
+// #ifdef TARGET_BMV2_V1MODEL
```

Comment:
Why keep these lines of code, they don't do anything>

Reply:
Removed